src/tracker/inflights.rs

```diff
@@ -35,6 +38,7 @@ impl Clone for Inflights {
             start: self.start,
             count: self.count,
             buffer,
+            cap: self.cap,
```

Comment:
L36 can be also removed.

Reply:
Then, I think we can use derive(Clone) for inflight

Reply:
Now that capacity is maintained explicitly, so the statement is not valid anymore. `derive(Clone)` is a good idea!.